onnx2kerastl/operation_layers.py

```diff
@@ -414,3 +407,8 @@ def target_layer(x, axis=axis, keepdims=keepdims):
     lambda_layer = keras.layers.Lambda(target_layer, name=keras_name)
     layers[node_name] = lambda_layer(input_0)
     lambda_func[keras_name] = target_layer
+
+
+def convert_reciprocal(node, params, layers, lambda_func, node_name, keras_name):
+    input_0 = ensure_tf_type(layers[node.input[0]],  name="%s_const" % keras_name)
+    layers[node_name] = tf.math.reciprocal(input_0)
```

Comment:
This operation is converted to TFOpLambda also?

Reply:
converted to 1/x